### PR TITLE
Fix for some grammar and spelling

### PR DESCRIPTION
**Pull request by suchorski**
- [x] documentation is added or updated

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -71,9 +71,9 @@ guidelines:
     examples of our style. Run "make doc-nits" to make sure that your
     documentation changes are clean.
 
- 7. For user visible changes (API changes, behaviour changes, ...),
+ 7. For user visible changes (API changes, behavior changes, ...),
```
Comment (paulidale) [inline CONTRIBUTING.md:74]: The project generally prefers UK spellings over USA.

File: NOTES-ANDROID.md [hunk 2]
```diff
@@ -59,16 +59,16 @@ Notes for Android platforms
...
  Also, PATH would need even further adjustment to cover unprefixed, yet
  target-specific, ar and ranlib. It's possible that you don't need to
  bother, if binutils-multiarch is installed on your Linux system.
 
  Another option is to create so called "standalone toolchain" tailored
  for single specific platform including Android API level, and assign its
- location to `ANDROID_NDK_ROOT`. In such case you have to pass matching
- target name to Configure and shouldn't use `-D__ANDROID_API__=N`. `PATH`
+ location to `ANDROID_NDK_ROOT`. In such case, you have to pass matching
+ target name to configure and shouldn't use `-D__ANDROID_API__=N`. `PATH`
```
Comment (paulidale) [inline NOTES-ANDROID.md:71]: The script's name is `Configure` with a capital.  This change is therefore incorrect.

**Review by paulidale — CHANGES_REQUESTED** [review]

**Comment by t8m** [comment]
The `CLA: trivial` needs to be placed in the commit message body (i.e. after an empty line after the commit message subject) and not in the subject.

**Comment by t8m** [comment]
I suppose this would be acceptable with CLA: trivial although the patch is not small.

**Comment by suchorski** [comment]
> The `CLA: trivial` needs to be placed in the commit message body (i.e. after an empty line after the commit message subject) and not in the subject.

How to fix the commit messaege? I forgot the empty line

**Comment by suchorski** [comment]
Making another pull request.

**Comment by t8m** [comment]
> How to fix the commit messaege? I forgot the empty line.

`git commit --amend`
and then
`git push --force`

**Pull request closed without merging**